spec/change_set_persisters/change_set_persister_spec.rb

```diff
@@ -1517,4 +1517,107 @@
       expect(reloaded.numismatic_artist_ids).to eq [output.id]
     end
   end
+
+  describe "preservation" do
+    context "when completing a `cloud` preservation_policy resource" do
+      it "saves to a backup location" do
+        file = fixture_file_upload("files/example.tif", "image/tiff")
+        resource = FactoryBot.create_for_repository(:pending_scanned_resource, preservation_policy: "cloud", files: [file])
+        change_set = DynamicChangeSet.new(resource)
+        change_set.validate(state: "complete")
+
+        output = change_set_persister.save(change_set: change_set)
+        expect(Wayfinder.for(output).preservation_object.metadata_node).not_to be_blank
```

Comment:
I'm nervous about testing that things aren't blank ever since that update_id bug. Is there something more specific we can check? Also see `be_blank` in other places in this spec.